src/main/java/dev/tycho/stonks/managers/DatabaseHelper.java

```diff
@@ -97,20 +102,15 @@ public void openInvitesList(Player player) {
     Stonks.newChain()
         .asyncFirst(() -> {
           List<Member> invites;
-          try {
-            invites = databaseManager.getMemberDao().getInvites(player);
-          } catch (SQLException e) {
-            e.printStackTrace();
-            return null;
-          }
+          invites = databaseManager.getMemberDao().getInvites(player);
           if (invites == null) {
-            sendMessage(player, "You don't have any pending invites!");
+            player.sendMessage(ChatColor.RED + "You don't have any invites!");
```

Comment:
Revert this

Reply:
https://github.com/schlatt-co/stonks/blob/a34c87a8a16fdce21ba0462044aedb07a35af01b/src/main/java/dev/tycho/stonks/managers/DatabaseHelper.java#L106